lib/random.gd

```diff
@@ -74,47 +86,46 @@ DeclareCategory( "IsRandomSource", IsComponentObjectRep );
 DeclareOperation( "Random", [IsRandomSource, IsInt, IsInt] );
 
+
 #############################################################################
 ##  
-#O  State( <rs> ) . . . . . . . . . . . . . . . state of random source
-#O  Reset( <rs> )
-#O  Reset( <rs>, <seed> ) . . . . . . . . . . . reset a random source
-#O  Init( <rs> )
-#O  Init( <prers>, <seed> )  . . . . . . . initialize a random source
+#O  State( <rs> )  . . . . . . . . . . . . . . . . . . state of random source
+#O  Reset( <rs>[, <seed>] )  . . . . . . . . . . . . .  reset a random source
 ##
 ##  <#GAPDoc Label="State">
 ##  <ManSection>
+##  <Heading>State and Reset for Random Sources</Heading>
 ##  <Oper Name="State" Arg='rs'/>
 ##  <Oper Name="Reset" Arg='rs[, seed]'/>
-##  <Oper Name="Init" Arg='prers[, seed]'/>
 ##
 ##  <Description>
-##  These are the basic operations for which random sources (see
-##  <Ref Filt="IsRandomSource"/>) must have methods. 
-##  <P/>
-##  <Ref Oper="State"/> should return a data structure which allows to recover the state
-##  of the random source such that a sequence of random calls using this 
-##  random source can be reproduced. If a random source cannot be reset 
-##  (say, it uses truly random physical data) then <Ref Oper="State"/>
-##  should return  <K>fail</K>.
+##  These are the basic operations for random sources (see
+##  <Ref Filt="IsRandomSource"/>).
 ##  <P/>
-##  <C>Reset( <A>rs</A>, <A>seed</A> )</C> resets the random source <A>rs</A> to a state described
-##  by <A>seed</A>, if the random source can be reset (otherwise it should do
-##  nothing). Here <A>seed</A> can be an output of <Ref Oper="State"/> and then should reset
-##  to that state. Also, the methods should always allow integers as <A>seed</A>.
-##  Without the <A>seed</A> argument the default <M><A>seed</A> = 1</M> is used.
+##  <Ref Oper="State"/> returns a data structure which admits recovering
+##  the state of the random source such that a sequence of random calls
+##  using this random source can be reproduced.
+##  If a random source cannot be reset (say, it uses truly random physical
+##  data) then <Ref Oper="State"/> returns <K>fail</K>.
 ##  <P/>
-##  <Ref Oper="Init"/> is the constructor of a random source, it gets an empty component 
-##  object <A>prers</A> which has already the correct type and should fill in the actual 
-##  data which are needed. Optionally, it should allow one to specify a 
-##  <A>seed</A> for the initial state, as explained for <Ref Oper="Reset"/>.
+##  <C>Reset( <A>rs</A>, <A>seed</A> )</C> resets the random source <A>rs</A>
+##  to a state described by <A>seed</A>, if the random source can be reset;
+##  otherwise it does nothing.
+##  Here <A>seed</A> can be an output of <Ref Oper="State"/> and then
+##  <A>rs</A> gets reset to that state.
+##  Also integers are allowed as <A>seed</A>.
+##  Without the <A>seed</A> argument the default <M><A>seed</A> = 1</M> is
+##  used.
```

Comment:
How about changing this as follows, and not making any claims about what the default seed is?
```suggestion
##  Without the <A>seed</A> argument a fixed default seed is used.
```

Reply:
O.k.